web/gds-admin-ui/src/pages/app/dashboard/TasksChart.js

```diff
@@ -0,0 +1,182 @@
+// @flow
+import React from 'react';
+import { Bar } from 'react-chartjs-2';
+import { Card } from 'react-bootstrap';
+import { useSelector } from 'react-redux';
+import { STACKED_BAR_LABEL } from '../../../constants/dashboard';
+import { formatLabelWithUnderscore } from '../../../utils';
+
+
+const barChartOpts = {
+    maintainAspectRatio: false,
+    legend: {
+        display: true,
+    },
+    tooltips: {
+        intersect: false,
+    },
+    hover: {
+        intersect: true,
+    },
+    plugins: {
+        filler: {
+            propagate: false,
+        },
+    },
+    scales: {
+        xAxes: [
+            {
+                reverse: true,
+                gridLines: {
+                    color: 'rgba(0,0,0,0.05)',
+                },
+                stacked: true
+            },
+        ],
+        yAxes: [
+            {
+                ticks: {
+                    stepSize: 10,
+                    display: false,
+                },
+                min: 10,
+                max: 100,
+                display: true,
+                borderDash: [5, 5],
+                gridLines: {
+                    color: 'rgba(0,0,0,0)',
+                    fontColor: '#fff',
+                },
+                stacked: true
+            },
+        ],
+    },
+};
+
+
+const TasksChart = (): React$Element<any> => {
+    const { reviews, isLoading } = useSelector(state => ({
+        reviews: state.Reviews.data,
+        isLoading: state.Reviews.loading
+    }))
+
+    const getWeeks = () => {
+        if (reviews && Array.isArray(reviews)) {
+            return reviews.map(review => review.week)
+        }
+
+        return []
+    }
+
+    const getData = (key = '') => {
+        if (reviews && Array.isArray(reviews)) {
+            return reviews.map(review => {
+                return review.registrations[key];
+            })
+        }
+
+        return []
+    }
+
+    const barChartData = {
+        labels: [
+            ...getWeeks()
+        ],
+        datasets: [
+            {
+                barPercentage: 0.7,
+                categoryPercentage: 0.7,
+                label: STACKED_BAR_LABEL.SUBMITTED,
+                backgroundColor: '#727cf5',
+                borderColor: '#727cf5',
+                data: getData(STACKED_BAR_LABEL.SUBMITTED),
+            },
+            {
+                barPercentage: 0.7,
+                categoryPercentage: 0.7,
+                label: formatLabelWithUnderscore(STACKED_BAR_LABEL.EMAIL_VERIFIED),
+                backgroundColor: '#1abc9c',
+                borderColor: '#1abc9c',
+                data: getData(STACKED_BAR_LABEL.EMAIL_VERIFIED),
```

Comment:
This would then become:

```javascript
              {
                barPercentage: 0.7,
                categoryPercentage: 0.7,
                label: formatLabelWithUnderscore(STACKED_BAR_LABEL.EMAIL_VERIFIED),
                backgroundColor: '#1abc9c',
                borderColor: '#1abc9c',
                data: getData(Status.EMAIL_VERIFIED),
              },
```

Reply:
Why this line ?
```javascript 
export const STACKED_BAR_LABEL = StatusLabel
```